src/txn/txn.c

```diff
@@ -890,6 +890,14 @@ __txn_commit_timestamps_usage_check(WT_SESSION_IMPL *session, WT_TXN_OP *op, WT_
           __wt_timestamp_to_string(op_ts, ts_string[0]),
           __wt_timestamp_to_string(prev_op_durable_ts, ts_string[1])));
 
+    if (FLD_ISSET(ts_flags, WT_DHANDLE_TS_ORDERED) && prev_op_durable_ts > 0 && !txn_has_ts)
```

Comment:
We use WT_TS_NONE for 0 timestamp.

Reply:
I would us prev_op_durable_ts != WT_TS_NONE in this case.

Reply:
Done